pom.xml

```diff
@@ -10,12 +10,12 @@
     </parent>
     <groupId>org.isf</groupId>
     <artifactId>openhospital-api</artifactId>
-    <version>0.0.2</version>
+    <version>1.12.0-SNAPSHOT</version>
```

Comment:
Not sure if we really need to change this... I think release 1.0 will be at the first release including this module (or anyway we cannot state now that it will with release 1.12)

Reply:
OK; reverted the change

Reply:
Alright! Thanks!